### PR TITLE
Fix potential issue in long_video_demo.py

### PR DESCRIPTION
**Pull request by dreamerlin**
This PR fix 2 potential issue in long_video_demo.py
1. The last of some videos will be decoded as None in OpenCV. When it is the case, demo will skip it.
2. Popleft only once for keeping img shape.

### CODE REVIEW

File: demo/long_video_demo.py [hunk 1]
```diff
@@ -70,6 +71,9 @@ def show_results():
         ind += 1
         prog_bar.update()
         ret, frame = cap.read()
+        if frame is None and ind == num_frames:
```
Comment (innerlee) [inline demo/long_video_demo.py:74]: why not just drop it?

Comment (innerlee) [reply demo/long_video_demo.py:74]: you know, things in videos are always inexact

File: demo/long_video_demo.py [hunk 2]
```diff
@@ -71,9 +71,9 @@ def show_results():
         ind += 1
         prog_bar.update()
         ret, frame = cap.read()
-        if frame is None and ind == num_frames:
-            warnings.warn('The last frame of the video is None.')
-            break
+        if frame is None:
+            warnings.warn('A certain frame of the video is None.')
```
Comment (innerlee) [inline demo/long_video_demo.py:75]: no warning needed

File: demo/long_video_demo.py (hunk 1 above)
Comment (dreamerlin) [reply demo/long_video_demo.py]: Indeed

**Review by innerlee — APPROVED** [review]

**Pull request merged**